packagingbuild/Dockerfile.debian

```diff
@@ -21,6 +33,10 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
         dpkg-buildpackage -b -uc -us && dpkg -i ../dh-virtualenv_*.deb && \
           rm -rf /tmp/dh-virtualenv*
 
-{%- if version in ('xenial') %}
+{%- if version in ('bionic') %}
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata
```

Comment:
We need to install ``tzdata`` package on Bionic otherwise packages tests fail the error bellow and services which rely on tz configuration (e.g. notifer) don't start.

```bash
2018-11-05 09:47:20,101 140063348096496 INFO (unknown file) [-] (PID=8739) Actions notifier started.
2018-11-05 09:47:20,103 140063348096496 ERROR (unknown file) [-] (PID=8739) Results tracker quit due to exception.
Traceback (most recent call last):
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/st2actions/cmd/st2notifier.py", line 58, in main
    return _run_worker()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/st2actions/cmd/st2notifier.py", line 37, in _run_worker
    actions_rescheduler = scheduler.get_rescheduler()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/st2actions/notifier/scheduler.py", line 40, in get_rescheduler
    timer = BlockingScheduler()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/apscheduler/schedulers/base.py", line 83, in __init__
    self.configure(gconfig, **options)
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/apscheduler/schedulers/base.py", line 122, in configure
    self._configure(config)
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/apscheduler/schedulers/base.py", line 694, in _configure
    self.timezone = astimezone(config.pop('timezone', None)) or get_localzone()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/tzlocal/unix.py", line 131, in get_localzone
    _cache_tz = _get_localzone()
  File "/opt/stackstorm/st2/lib/python3.6/site-packages/tzlocal/unix.py", line 125, in _get_localzone
    raise pytz.UnknownTimeZoneError('Can not find any timezone configuration')
pytz.exceptions.UnknownTimeZoneError: 'Can not find any timezone configuration'
```

It's interesting we need to do this on Bionic but not on Xenial. Perhaps tzdata is already included on Xenian by default or installed via some other dependency?